### ISSUE TITLE
JAR File Works Only With java -jar

### ISSUE THREAD

**Issue by snoidetx** [nus-cs2103-AY2122S2/forum#159]
I am trying to test the portability of my app. When I download a release and put it into a folder, if I double click the file, it cannot create a new directory for the logs file (which is used in saving and loading). However, if I `cd` to the directory and `java -jar FILENAME`, the file is created normally. Is this an issue that I should care about? If yes, how can I fix it?

**Comment by damithc** [comment]
@snoidetx In case this is OS-dependent, it might help to state which OS are you using

**Comment by snoidetx** [comment]
@damithc I am using MacOS Monterey :)

**Comment by DALIN-Prog** [comment]
Hi, I have also faced this issue and the last comment in this [link](https://github.com/nus-cs2103-AY2122S1/forum/issues/221) helps (link quoted from @damithc)

**Comment by snoidetx** [comment]
Thanks @DALIN-Prog ! I have managed to get it works. For those Mac users who might also face this issue, I have compiled the solution here:

1. Go to System Preference - Security & Privacy - Privacy - Full Disk Access, click "+" and add `/System/Library/CoreServices/JavaLauncher`.
2. Try again and it should work now! 
 
Hence the probable reason is that the jar file does not full disk access and it cannot write the folder.

Credit: [here](https://discussions.apple.com/thread/252709578)
